Controllers/NewTodoController.cs

```diff
@@ -0,0 +1,69 @@
+using Microsoft.AspNetCore.Mvc;
+using TodoNET6.Models;
+using TodoNET6.Services;
+
+namespace TodoNET6.Controllers
+{
+    [ApiController]
+    [Route("task")]
+    public class NewTodoController : ControllerBase
+    {
+        private readonly ITodoRepository _todoRepository;
+
+        public NewTodoController(
+            ITodoRepository todoRepository
+        )
+        {
+            _todoRepository = todoRepository;
+        }
+
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Todo>>> GetTodos()
+        {
+            var results = await _todoRepository
+                .WhereAsync(todo => true);
+
+            return results.ToList();
+        }
+
+        [HttpGet("{uuid}")]
+        public async Task<ActionResult<Todo>> GetTodo(Guid uuid)
+        {
+            var todoFromDb = await _todoRepository.FindAsync(uuid);
+
+            if (todoFromDb == null)
+            {
+                return NotFound();
+            }
+
+            return todoFromDb;
+        }
+
+        [HttpPost]
+        public async Task<ActionResult<Todo>> CreateTodo([FromBody]Todo todo)
+        {
+            var createdTodo = await _todoRepository.CreateAsync(todo);
+
+            return createdTodo;
+        }
+
+        [HttpPut("{uuid}")]
+        public async Task<ActionResult<Todo>> UpdateTodo(
+            Guid uuid, 
+            [FromBody] Todo todoToUpdate
```

Comment:
So, I had not been using [FromBody] thus far, where we would be explicitly saying I am getting a Todo, for example, from a request body. In postman, I had not run into trouble, but can you give me a case where we would have to use [FromBody] or [FromUri]? I feel like I understand why you may want to use it but not grasping when you would have to use it.